src/new-relic/azext_new_relic/aaz/latest/new_relic/monitor/_create.py

```diff
@@ -51,6 +51,8 @@ def _build_arguments_schema(cls, *args, **kwargs):
             required=True,
         )
         _args_schema.resource_group = AAZResourceGroupNameArg(
+            options=["--resource-group"],
```

Comment:
It seems this change introduces a breaking change (Previously it supported both "--resource-group" and "--g", but now it only supports "--resource-group"). Please check other breaking changes as well.

Reply:
This is not expected. I did it by mistake will revert this change.

Reply:
@jsntcy I have reverted the changes which were breaking changes. Please have a look if anything else is breaking.

Reply:
@dipeshbhakat-microsoft still two?

![image](https://github.com/Azure/azure-cli-extensions/assets/12371639/7da8e1ff-78e4-4d50-9ba8-a004da04954b)

Reply:
I have now added the remaining test cases and fixed the breaking changes

Reply:
@necusjz all work from my side is done now. Let me know if anything else is required from my side.